packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -115,7 +115,7 @@ export default function useTabNav() {
 			// these are not rendered in the content and perhaps in the
 			// future they can be rendered in an iframe or shadow DOM.
 			if (
-				( isFormElement( event.target ) ||
+				( isFormElement( event.target, true ) ||
 					event.target.getAttribute( 'data-block' ) ===
 						getSelectedBlockClientId() ) &&
 				isFormElement( focus.tabbable[ direction ]( event.target ) )
```

Comment:
Should this call to `isFormElement` also check for contenteditables? 

It looks like it says 'if the current element is a form element, content editable or a block boundary, and the next element is a form element, don't do anything'. But I'm not sure what doing nothing achieves.

Reply:
No, it should not. Adding true here will find another block on next Tab key press vs. jumping to the sidebar.

Reply:
@talldan I could not make this work again when I restarted my dev environment so I removed the changes. I'll try again in another PR. This whole writing flow feature still needs some work for screen reader users.